Makefile

```diff
@@ -1,8 +1,16 @@
+VERSION=0.0.8
```

Comment:
I might move this into a file, `VERSION.txt`, so it can be consumed by other things potentially. Looks good for now.